source/priority-queue.ts

```diff
@@ -0,0 +1,44 @@
+import {Queue, RunFunction} from './queue';
+import lowerBound from './lower-bound';
+import {QueueAddOptions} from './options';
+
+export interface PriorityQueueOptions extends QueueAddOptions {
+	priority: number;
+}
+
+export default class PriorityQueue implements Queue<PriorityQueueOptions> {
+	private readonly queue: (PriorityQueueOptions & { run: RunFunction })[];
```

Comment:
Can't you set the initial value here instead of in the constructor?

Reply:
Certainly! But then we should be consistent and also do it in the ones from [here](https://github.com/sindresorhus/p-queue/blob/edd0356229d0ca4a2b22c191ddac0b8aa628dbd1/source/index.ts#L67) that aren't decided upon at runtime as well